consensus/src/experimental/buffer_manager.rs

```diff
@@ -291,12 +320,6 @@ impl BufferManager {
                 return;
             }
         };
-        // if this batch of blocks are all suffix blocks (reconfiguration block is in one of previous batch)
-        // we pause the execution from here and wait for the reconfiguration to be committed.
-        if executed_blocks.first().unwrap().is_reconfiguration_suffix() {
```

Comment:
Can you explain why this is not needed anymore?

Reply:
it's described here https://aptos-org.slack.com/archives/C03BDV9NVGA/p1658256672671459

Reply:
Can you add the description in the PR as well for future reference?